core/app/models/spree/variant.rb

```diff
@@ -183,7 +183,7 @@ def is_backorderable?
     # @return [String] a sentence-ified string of option values.
     def options_text
       values = option_values.includes(:option_type).sort_by do |option_value|
-        option_value.option_type.position
+        option_value.option_type&.position
```

Comment:
Doesn't `sort_by` raise an error 'cause you're comparing `nil` with other values ?

I know we have a lot of these in our codebase where we need to fake a value to avoir that error. Maybe it's not with `sort_by` and just `sort` 🤷‍♂️  but better safe than sorry and ask.

Reply:
@stem Based on the error message, the sort_by doesn't raise the error.

Reply:
If you have an error message, it would be great to share it. If someone has the same message and google it, finding this issue could be useful.